src/CMakeLists.txt

```diff
@@ -184,12 +195,14 @@ macro(automaton_test test_path test_name)
     automaton/tests/${test_path}/${test_name}.cc
     automaton/tests/data/proto_files.cc
   )
+  target_include_directories(${test_name} PRIVATE ${GTEST_INCLUDE_DIR})
   target_link_libraries(${test_name} automaton-core ${GTEST_LIBS})
   add_test(
     NAME ${test_name}
     COMMAND ${test_name}
     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
   )
+  set_property(TARGET ${test_name} PROPERTY VS_DEBUGGER_WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
Need to add this for some of the other targets like the automaton-core-cli.
(And the above WORKING_DIRECTORY parameter)
Might be best to create a custom target macro that creates an easy to debug target which also links statically all the libs. (as I understand that's already the case though - the static linking)

Reply:
Macro added